.cirrus.yml

```diff
@@ -688,7 +689,7 @@ upgrade_test_task:
     name: "Upgrade test: from $PODMAN_UPGRADE_FROM"
     alias: upgrade_test
     skip: *tags
-    only_if: $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' || $CIRRUS_CRON != ''
+    only_if: $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' && $CIRRUS_CHANGE_TITLE !=~ '.*CI:BUILD.*' && $CIRRUS_CRON != ''
```

Comment:
`only_if: $CIRRUS_CRON != ''`

Reply:
Upgrade? AFAIK that should run on all PRs. It's very quick anyway.

Reply:
Oh, okay.